### ISSUE TITLE
Anchor 0.30.1 build fails at pyth-solana-receiver-sdk 0.3.0 version

### ISSUE THREAD

**Issue by ouoabcde** [pyth-network/pyth-crosschain#1759]
I'm using Anchor 0.30.1 version. I installed pyth-solana-receiver-sdk 0.3.0 version but build fails with this kind of errors.
```
error[E0599]: no function or associated item named `get_full_path` found for struct `MerklePriceUpdate` in the current scope
error[E0599]: no function or associated item named `create_type` found for struct `MerklePriceUpdate` in the current scope
error[E0599]: no function or associated item named `insert_types` found for struct `MerklePriceUpdate` in the current scope
```
Here is the full [error log](https://github.com/user-attachments/files/16139572/error.log).

**Comment by lopeselio** [comment]
so using borsh instead of serde_with for the purpose will solve this issue?

**Comment by mubarizkyc** [comment]
https://github.com/mubarizkyc/solpyth

**Comment by mubarizkyc** [comment]
@lopeselio  use borsh instead of anchor se/de

**Comment by mubarizkyc** [comment]
https://github.com/mubarizkyc/pyth-crosschain/tree/fixed-seraialization

**Comment by guibescos** [comment]
I like your fix @mubarizkyc , this will allow Anchor 0.30.1 to use the latest version of the sdk.
Another option is to use `pyth-solana-receiver-sdk` 0.2.0

**Comment by guibescos** [comment]
Should be fixed in  `pyth-solana-receiver-sdk = "0.3.1".`
